component/main.jsonnet

```diff
@@ -5,6 +5,22 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.cert_exporter;
 
+local namespace = kube.Namespace(params.namespace) {
+  metadata+: {
+    annotations+: {
+      'openshift.io/node-selector': '',
+    },
+    labels+: {
+      'app.kubernetes.io/name': params.namespace,
+      'topolvm.cybozu.com/webhook': 'ignore',
```

Comment:
I don't see how a topolvm label has any business being present in this component.